pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+
+from pootle_language.models import Language
+from pootle_project.models import Project
+
+
+class Command(BaseCommand):
+    help = "Creates a new project from command line."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--code',
+            action='store',
+            dest="code",
+            help='Project code',
+        )
+        parser.add_argument(
+            "--name",
+            action="store",
+            dest="name",
+            help="Project name",
+        )
+        parser.add_argument(
+            "--quality",
+            action="store",
+            dest="quality",
+            default="standard",
+            help="Quality management",
+        )
+        parser.add_argument(
+            "--filetype",
+            action="store",
+            dest="filetype",
+            help="File type: [ po/pot | xliff | xlf | ts | lang ]",
```

Comment:
`po/pot` might be difficult to type. I suggest to use just `po`.

Reply:
these need to match `filetype` names

Reply:
If I am correct these are https://github.com/translate/pootle/blob/5228e3d89df31492b7271836f1f494edc4b47d62/pootle/apps/pootle_format/default.py#L10-L30

Reply:
there is a special api for getting these, they can be registered by 3rd party software (might be same for presets actually - i need to check)